packages/buildbucket-dart/lib/buildbucket_proto.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+library buildbucket_proto;
+
+export 'src/generated/go.chromium.org/luci/buildbucket/proto/build.pb.dart' show Build;
```

Comment:
Is the plan to export only what we need?

Reply:
For now yes, but we can expand it as needed.